Docker/FreshRSS.Apache.conf

```
@@ -29,6 +47,12 @@ TraceEnable Off
 
 <Directory /var/www/FreshRSS/p/i>
 	IncludeOptional /var/www/FreshRSS/p/i/.htaccess
+	ExpiresActive Off
+
+	<IfDefine OIDC_ENABLED>
+		AuthType openid-connect
+		Require valid-user
+	</IfDefine>
```

```suggestion
	ExpiresActive Off

	<IfDefine OIDC_ENABLED>
		AuthType openid-connect
		Require valid-user
	</IfDefine>
	IncludeOptional /var/www/FreshRSS/p/i/.htaccess
```

Let's move any user `.htaccess` down to allow custom override